### PR TITLE
Added new PR comments summary design

### PR DESCRIPTION
**Pull request by NimrodGolan**
#376 

### Description

> New PR comments summary design is implemented for all supported scanners (SAST, SCA & AST)

### References

> https://dev.azure.com/CxFlow/CxFlow/_workitems/edit/233

### CODE REVIEW

**Review by ofersk — APPROVED** [review]

**Review by ofersk — APPROVED** [review]

File: src/main/java/com/checkmarx/flow/utils/MarkDownHelper.java [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+
+    private static final String SUMMARY_SUFFIX = " Summary";
+    public static final String SAST_SUMMARY_HEADER = CX_PREFIX + SAST_SCANNER + SUMMARY_SUFFIX;
+    static final String AST_SUMMARY_HEADER = CX_PREFIX + AST_SAST_SCANNER + SUMMARY_SUFFIX;
+    static final String SCA_SUMMARY_HEADER = CX_PREFIX + SCA_SCANNER + SUMMARY_SUFFIX;
+
+    static final String LINE_BREAK = "<br />";
+    static final String NBSP = "&nbsp;";
+
+    private static final String IMAGE_TEMPLATE = "![%s](%s)";
```
Comment (AvivCX) [inline src/main/java/com/checkmarx/flow/utils/MarkDownHelper.java:29]: @NimrodGolan , @ofersk
Why does the image template need to have the url suffix ( '(%s)' )
it makes the icon be a link - and it doesn't seem to be right

Comment (ofersk) [reply src/main/java/com/checkmarx/flow/utils/MarkDownHelper.java:29]: in MD you add an image using this template:
`alt text`

in GitHub there is a non standard way, to add a local image using:
`[[url/to/img.png]]`

Comment (NimrodGolan) [reply src/main/java/com/checkmarx/flow/utils/MarkDownHelper.java:29]: @AvivCX - this is a basic MD for loading an image. where do you see this bug?

Comment (AvivCX) [reply src/main/java/com/checkmarx/flow/utils/MarkDownHelper.java:29]: in the current shape you can press it like a button, and it opens a separate page with the icon.
I dont think it is purposed

Comment (ofersk) [reply src/main/java/com/checkmarx/flow/utils/MarkDownHelper.java:29]: @NimrodGolan  in GitHub append ?raw=true to the image url will make the trick.
ex: ``

this is a GitHub "feature"

**Pull request merged**